### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Zen/Activities/MainActivity.cs [hunk 1]
```diff
@@ -63,17 +65,13 @@ protected override void OnActivityResult(int requestCode, [GeneratedEnum] Result
...
                 Task.Run(() =>
                 {
-                    while (true)
+                    while (!isFinish)
                     {
-                        try
+                        if (FindViewById<TextView>(Resource.Id.textViewTime) is TextView textViewTime)
                         {
-                            FindViewById<TextView>(Resource.Id.textViewTime).Text = minute;
+                            textViewTime.Text = minute;
```
Comment (voquanghoa) [inline Zen/Activities/MainActivity.cs]: Add `Thread.Sleep(200);` here. 
`200` is 200 milliseconds (2/10s), change it if it's necessary

File: Zen/Activities/MainActivity.cs [hunk 2]
```diff
@@ -118,8 +116,14 @@ private void DisplayFragment(Fragment fragment)
         private void BottomNavigation_NavigationItemSelected(object sender, BottomNavigationView.NavigationItemSelectedEventArgs e)
         {
             LoadFragment(e.Item.ItemId);
+        }
+
+        public override void Finish()
+        {
+            isFinish = true;
+
```
Comment (voquanghoa) [inline Zen/Activities/MainActivity.cs]: Add `Thread.Sleep(100);` here to prevent error

File: Zen/Activities/MainActivity.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Zen/Activities/MainActivity.cs]: Done

File: Zen/Activities/MainActivity.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Zen/Activities/MainActivity.cs]: Done

**Pull request merged**